readme.md

```diff
@@ -217,6 +217,7 @@
 - [Preact](https://github.com/ooade/awesome-preact) - App framework.
 - [Progressive Enhancement](https://github.com/jbmoelker/progressive-enhancement-resources)
 - [Next.js](https://github.com/unicodeveloper/awesome-nextjs) - Framework for server-rendered React apps.
+- [Hyperapp](https://github.com/hyperapp/awesome-hyperapp) - 1 KB JavaScript library for building frontend applications.
```

Comment:
Change the description to:

> Tiny JavaScript library for building front-end apps.

Reply:
@sindresorhus Shouldn't it be "App framework"?

Reply:
I only used "App framework" out of laziness.

Reply:
@sindresorhus Is this supposed to be the category of the software or the software description?

If category, then "JavaScript Frontend Framework" works.

Otherwise, why come up with a different description than the "official" one?

Reply:
Description. I make my own descriptions as most official descriptions are terrible or sub-optimal. In this case, it was fine, but I just wanted to tweak it a little bit.

Reply:
Thanks, but I prefer the current description, as it's the same we're using everywhere else. Front-end works also, but [both](https://en.wiktionary.org/wiki/front-end) are [valid](https://en.wiktionary.org/wiki/frontend) it seems.

Reply:
Ping @sindresorhus